tamr_client/instance/backup.py

```diff
@@ -0,0 +1,94 @@
+from tamr_client import response
+from tamr_client._types import Instance, Session, URL
+from tamr_client.exception import TamrClientException
+
+
+class AlreadyCanceled(TamrClientException):
+    """Raised when attempting to cancel a backup that has already failed or been canceled.
+    """
+
+    pass
+
+
+class NotFound(TamrClientException):
+    """Raised when referencing a backup that does not exist on the server.
+    """
+
+    pass
+
+
+def get(session: Session, instance: Instance) -> list:
+    """Get information on Tamr backups.
+
+    Args:
+        session: Tamr session
+        instance: Tamr instance
+
+    Returns:
+        A list of Tamr backups
+    """
+    url = URL(instance=instance, path='backups')
+    r = session.get(str(url))
+    if r.status_code == 404:
+        raise NotFound(str(url))
+    return response.successful(r).json()
+
+
+def get_by_id(session: Session, instance: Instance, backup_id: str) -> dict:
+    """Get information on a specific Tamr backup.
+
+    Args:
+        session: Tamr session
+        instance: Tamr instance
+        backup_id: Relative backup ID
+
+    Returns:
+        Information on specific backup
+
+    Raises:
+        NotFound: If no backup found at the specified URL
+    """
+    url = URL(instance=instance, path=f"backups/{backup_id}")
+    r = session.get(str(url))
+    if r.status_code == 404:
+        raise NotFound(str(url))
+    return response.successful(r).json()
+
+
+def initiate(session: Session, instance: Instance) -> dict:
+    """Initiate a Tamr backup.
+
+    Args:
+        session: Tamr session
+        instance: Tamr instance
+
+    Returns:
+        Information on initiated backup
+    """
+    url = URL(instance=instance, path="backups")
+    r = session.post(str(url))
+    return response.successful(r).json()
+
+
+def cancel(session: Session, instance: Instance, backup_id: str) -> dict:
```

Comment:
After we model backups as types, we should be operating on that type:

```suggestion
def cancel(session: Session, instance: Instance, backup: Backup) -> dict:
```